test/Autofac.Integration.AspNetCore.Multitenant.Test/TestDependencies/TestServerFixture.cs

```diff
@@ -87,6 +91,12 @@ public void Configure(IApplicationBuilder builder)
                         await context.Response.WriteAsync(whoAmI.Id);
                     });
 
+                    routeBuilder.MapGet("scoped-endpoint", async context =>
+                    {
+                        var scopedDep = context.RequestServices.GetRequiredService<IScopedDependency>();
+                        await context.Response.WriteAsync(scopedDep.GetHashCode().ToString());
```

Comment:
Have the `IScopedDependency` with a `Guid` property of `Id` and have `ScopedDependency` generate a `Guid.NewGuid()` for its ID during construction. Hash codes may be equal even for objects that aren't reference equal, so this could technically fail later.

Reply:
Addressed